backend/routes/admin_routes.py

```
@@ -251,3 +259,422 @@ def get_admin_info():
+        user_id = get_jwt_identity()
+        db_session = SessionLocal()
+
+        user = db_session.query(User).filter(User.id == user_id).first()
+        if not user:
+            return jsonify({"error": USER_NOT_FOUND}), 404
+
+        # Update Portale Alloggi credentials
+        if "portale_username" in data:
+            user.portale_username = data["portale_username"]
+
+        if "portale_password" in data and data["portale_password"]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)